doc/book/app_server/cookbook.rst

```diff
@@ -774,7 +774,92 @@ An "HTML" file for this server, including Lua, could look like this
     </body>
     </html>
 
+.. _cookbook-select-all-go:
 
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+select_all.go
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+In Go, there is no one-liner to select all tuples from a Tarantool space.
+Yet you can use a script like this one.
+
+.. code-block:: go
+
+    package main
+
+    import (
+    	"fmt"
+
+    	"github.com/tarantool/go-tarantool"
+    )
+
+    /*
```

Comment:
Maybe we should add some comment about this code? It should be called on instance we want to connect. Is it obvious?